### ISSUE TITLE
[BUG] -  can't open dropdown in production

### ISSUE THREAD

**Issue by nazrul-dev** [nextui-org/nextui#2768]
### NextUI Version

2.3.2

### Describe the bug

can't open drodpdown in production, and you can also test it on your website, it can't be opened, even though my application is live

![image](https://github.com/nextui-org/nextui/assets/76646247/d1062c05-2dac-4521-bd81-41084f34a0d4)

### Steps to Reproduce the Bug or Issue

1. go to https://nextui.org/docs/components/dropdown
2. click one of the demo dropdowns

### Expected behavior

it should be open

### Operating System Version

windows

### Browser

Chrome

**Comment by wingkwong** [comment]
preliminary findings: 
- failed in prod storybook & prod doc site local storybook
- works in local storybook & local doc site (`main` branch)

![image](https://github.com/nextui-org/nextui/assets/35857179/47e8b4bb-3b2e-4191-a891-030754e5e0c5)

![image](https://github.com/nextui-org/nextui/assets/35857179/d5849606-9309-4b86-95eb-c69602545036)

**Comment by YoFiq** [comment]
in my case, dropdown item ignores onPress/onClick events

**Comment by wingkwong** [comment]
@YoFiq onpress/onclick issue on dropdown has been addressed in 2.3.3. Are you using this version?

**Comment by YoFiq** [comment]
@wingkwong i used 2.3.2, and dropdown just ignored onPress/onClick events
on version 2.3.3 dropdown not displayed menu at all

downgraded to this versions and everything works fine
"@nextui-org/react": "2.2.10",
"framer-motion": "^10.2.4",

**Comment by YoFiq** [comment]
`export const DropdownTest = ({
	isCondition,
	handleTest1,
	handleTest2,
	handleTest3
}: Props) => {
	const menuOptions = useMemo(() => {
		const options = [
			{
				key: 'title_1',
				title: 'title 1',
				onPress: () => handleTest1()
			},
			{
				key: 'title_2',
				title: 'title 2',
				onPress: () => handleTest2()
			}
		];
		if (isCondition) {
			options.push({
				key: 'title_3',
				title: 'title 3',
				onPress: () => handleTest3()
			});
		}
		return options;
	}, [isCondition, handleTest1, handleTest2, handleTest3]);

	const getOptionsInProgressItems = () => {
		const dropdownChildren = menuOptions.map((menu) => (
			<DropdownItem key={menu.key} onPress={menu.onPress}>
				{menu.title}
			</DropdownItem>
		));
		return { children: dropdownChildren };
	};

	const dropdownChildren = getOptionsInProgressItems();

	return (
		<div className="flex flex-row items-center justify-between">
			<Dropdown
				classNames={{
					content: 'bg-lightUI-6 text-lightUI-1 rounded-lg',
					trigger: 'z-0 outline-none'
				}}
			>
				<DropdownTrigger>
					<div className="flex h-10 min-h-10 w-10 min-w-10 items-center justify-center rounded-lg bg-lightUI-9 2xl:h-11 2xl:w-11">
						<MoreVert height={20} width={20} />
					</div>
				</DropdownTrigger>
				<DropdownMenu
					classNames={{
						base: 'p-0'
					}}
					aria-label="Active menu"
					itemClasses={{
						title: 'group-data-[hover=true]:text-blue-light-hovered ',
						base: '!text-lightUI-1 text-sm py-2 px-3 [&>svg]:data-[hover=true]:text-blue-light-hovered [&>span>svg]:!hidden font-poppins data-[hover=true]:bg-lightUI-8'
					}}
					{...dropdownChildren}
				/>
			</Dropdown>
		</div>
	);
};`

**Comment by YoFiq** [comment]
```
export const DropdownTest = ({isCondition, handleTest1, handleTest2, handleTest3}: Props) => {
	const menuOptions = useMemo(() => {
		const options = [
			{
				key: 'title_1',
				title: 'title 1',
				onPress: () => {
                                     handleTest1();
                                     handleTest2();
                                 }
			},
			{
				key: 'title_2',
				title: 'title 2',
				onPress: () => {
                                     handleTest2();
                                     handleTest3();
                                 }
			}
		];
		if (isCondition) {
			options.push({
				key: 'title_3',
				title: 'title 3',
				onPress: () => {
                                     handleTest1();
                                     handleTest3();
                                 }
			});
		}
		return options;
	}, [isCondition, handleTest1, handleTest2, handleTest3]);

	const getOptionsItems = () => {
		const dropdownChildren = menuOptions.map((menu) => (
			<DropdownItem key={menu.key} onPress={menu.onPress}>
				{menu.title}
			</DropdownItem>
		));
		return { children: dropdownChildren };
	};

	const dropdownChildren = getOptionsItems();`

	`return (
		<div className="flex flex-row items-center justify-between">
			<Dropdown
				classNames={{
					content: 'bg-lightUI-6 text-lightUI-1 rounded-lg',
					trigger: 'z-0 outline-none'
				}}
			>
				<DropdownTrigger>
					<div className="flex h-10 min-h-10 w-10 min-w-10 items-center justify-center rounded-lg bg-lightUI-9 2xl:h-11 2xl:w-11">
						<MoreVert height={20} width={20} />
					</div>
				</DropdownTrigger>
				<DropdownMenu
					classNames={{
						base: 'p-0'
					}}
					aria-label="Active menu"
					itemClasses={{
						title: 'group-data-[hover=true]:text-blue-light-hovered ',
						base: '!text-lightUI-1 text-sm py-2 px-3 [&>svg]:data-[hover=true]:text-blue-light-hovered [&>span>svg]:!hidden font-poppins data-[hover=true]:bg-lightUI-8'
					}}
					{...dropdownChildren}
				/>
			</Dropdown>
		</div>
	);
};
```

**Comment by wingkwong** [comment]
Fixed in [v2.3.4](https://github.com/nextui-org/nextui/releases/tag/%40nextui-org%2Freact%402.3.4)
